explorer/app/models/viewModels.ts

```diff
@@ -1,14 +1,14 @@
 //Project
-export interface ProjectViewModel {
-  json: string;
-  projectName: string;
+export interface DetailViewModel {
+  json?: string;
+  detailName?: string;
 }
 
-export interface ProjectItemViewModel {
+export interface ItemViewModel {
   uuid: string;
   projectName: string;
```

Comment:
These models will change as we add more detail but perhaps generalize this to just `name` for now?

Reply:
right! Good idea!

Reply:
I've also noticed that I had a folder called `projects`, I renamed it to `entity`